packages/valory/skills/task_submission_abci/behaviours.py

```diff
@@ -1204,7 +1214,7 @@ def _get_encoded_deliver_data(
                 self.context.logger.warning(
                     f"get_encoded_data_for_request unsuccessful!: {contract_api_msg}"
                 )
-                return None
+                return ()
```

Comment:
This will raise:

https://github.com/valory-xyz/mech-predict/blob/da0a7fafd66a94882514648d3e95cd0f69dfdfb0/packages/valory/skills/task_submission_abci/behaviours.py#L1268-L1273